src/Spryker/Zed/Shipment/Persistence/ShipmentPersistenceFactory.php

```diff
@@ -95,7 +96,15 @@ public function createShipmentMapper(): ShipmentMapper
      */
     public function createShipmentMethodMapper(): ShipmentMethodMapperInterface
     {
-        return new ShipmentMethodMapper();
+        return new ShipmentMethodMapper($this->createStoreRelationMapper());
```

Comment:
you don't need store relationship support, see https://github.com/spryker/shipment/blob/7.0.0/src/Spryker/Zed/Shipment/Persistence/ShipmentPersistenceFactory.php#L96